build.gradle

```diff
@@ -77,6 +77,7 @@ subprojects { project ->
 
   dependencies {
     compile spinnaker.dependency('groovy')
+    compile 'io.fabric8:kubernetes-api:2.2.70'
```

Comment:
Drop a todo here to move this here: https://github.com/spinnaker/spinnaker-dependencies/blob/master/src/spinnaker-dependencies.yml